netbox/netbox/navigation/menu.py

```diff
@@ -454,6 +454,17 @@
                 ),
             ),
         ),
+        MenuGroup(
+            label=_('Plugins'),
+            items=(
+                MenuItem(
+                    link='extras:plugin_list',
+                    link_text=_('Plugins'),
+                    permissions=['core.view_configrevision'],
```

Comment:
We don't have a permission for viewing plugins. It should be sufficient to set `staff_only=True` below.
```suggestion
```